Content.Shared/Inventory/InventoryTemplatePrototype.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Robust.Shared.Maths;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+using Robust.Shared.Utility;
+
+namespace Content.Shared.Inventory;
+
+[Prototype("inventoryTemplate")]
+public class InventoryTemplatePrototype : IPrototype
+{
+    [DataField("id", required: true)]
+    public string ID { get; } = string.Empty;
+
+    //todo this is supreme shit and ideally slots should be stored in a given equipmentslotscomponent on each equipment
+    [DataField("slots")]
+    public SlotDefinition[] Slots { get; } = Array.Empty<SlotDefinition>();
+}
+
+[DataDefinition]
+public class SlotDefinition
+{
+    [DataField("name", required: true)] public string Name { get; } = string.Empty;
+
+    [DataField("slotTexture")] public string TextureName { get; } = "pocket";
+
+    //todo paul is this how you serialize flags?
```

Comment:
seems to work?

Reply:
haven't tested multiple flags in a slot yet

Reply:
Didn't check the inventory prototype, but for the item component which defines the flags in the same way I was able to just edit the cigarette definition from
```yaml
Slots: [ mask ]
```
to
```yaml
Slots: [ mask, head ]
```
and it seemed to work.

Reply:
nice